Flora/Flora/View Controllers/SearchViewController.m

```diff
@@ -120,6 +120,8 @@ - (IBAction)onTapSearch:(id)sender {
 - (IBAction)onTapLocation:(id)sender {
     [self presentViewController:self.locationAlert animated:YES completion:nil];
 }
+- (IBAction)tappedUseLocation:(id)sender {
```

Comment:
in general.. I would try and leave OUT any changes to your code (except for the pod change) when you are adding a podfile.